File: specification/specs/x_pack/transform/stop_transform/StopTransformRequest.ts
@rest_spec_name('transform.stop_transform')
class StopTransformRequest extends RequestBase {
  path_parts?: {
    transform_id: Name
  }
  query_parameters?: {
    allow_no_match?: boolean
    force?: boolean
    timeout?: Time
    wait_for_checkpoint?: boolean
    wait_for_completion?: boolean
  }
  body?: {}
}

File: output/csharp/XPack/Transform/StopTransform/StopTransformRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class StopTransformRequest : RequestBase {
		
		[DataMember(Name="allow_no_match")]
		public bool AllowNoMatch { get; set; }


		[DataMember(Name="force")]
		public bool Force { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }


		[DataMember(Name="wait_for_checkpoint")]
		public bool WaitForCheckpoint { get; set; }


		[DataMember(Name="wait_for_completion")]
		public bool WaitForCompletion { get; set; }

	}
}
